index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+
+    <title>portfolio</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="bar1"> 9:41
+            <img src="./images/Group 4.png" alt="icons"></div>
```

Comment:
- [ OPTIONAL] Kindly note that this part is not necessary, you can remove it.
![image](https://user-images.githubusercontent.com/98453979/205262868-da1ba486-6d88-4e40-ac8b-c353445766b6.png)